README.md

```diff
@@ -120,7 +120,7 @@ add the library to the $libdir/plugins/ directory.
 	cd $libdir/plugins/
 	sudo ln -s ../pgtt.so
 
-Then it will be possible to use it using `session_preload_libraries = '$libdir/plugins/pgtt'` in postgresql.conf
+Then it will be possible to use it using `session_preload_libraries = 'pgtt'` in postgresql.conf
```

Comment:
this change is ok, but we should probably remote the whole part about non-superuser usage as an explicit `LOAD` is not required anymore.

@darold any objection?

Reply:
No objection, session_preload_libraries must be the prior way to use pgtt.

Reply:
@simonnagl do you want to take care of it?

Reply:
Hi, I cannot do it without trying things out. I do not know if installing into the `$libdir/plugins/`` directory is not needed anymore or just loading plugin as non superuser changes. Please feel free to commit to this pull request.

Reply:
Well, the regression tests are working this way.  I added the required changes.